app/src/ui/clone-repository/clone-repository.tsx

```diff
@@ -200,7 +216,11 @@ export class CloneRepository extends React.Component<
       this.updateUrl(initialURL)
     }
 
-    window.addEventListener('focus', this.onWindowFocus)
+    this.checkIsTopMostDialog(this.props.isTopMost)
+  }
+
+  public componentWillUnmount(): void {
+    window.removeEventListener('focus', this.onWindowFocus)
```

Comment:
Could/should we leverage the helper here as well instead of duplicating?

```suggestion
    this.checkIsTopMostDialog(false)
```